package.json

```diff
@@ -7,21 +7,21 @@
   "scripts": {
     "build": "npm run compile-sass && tsc && npm run copy-views",
     "clean": "rm -rf dist build node_modules stylesheets",
-    "compile-sass": "sass --quiet-deps --no-source-map --load-path=node_modules/govuk-frontend --load-path=node_modules/@ministryofjustice/frontend --load-path=. assets/scss/application.scss:./assets/stylesheets/application.css assets/scss/application-ie8.scss:./assets/stylesheets/application-ie8.css --style compressed",
+    "compile-sass": "npx sass --quiet-deps --no-source-map --load-path=node_modules/govuk-frontend --load-path=node_modules/@ministryofjustice/frontend --load-path=. assets/scss/application.scss:./assets/stylesheets/application.css assets/scss/application-ie8.scss:./assets/stylesheets/application-ie8.css --style compressed",
```

Comment:
Were these scripts not working for you without `npx`? Did you run `script/setup` before using these?

Reply:
That's correct, though it's on the MoJ macbook. The other reason is npm by default will use the global version if installed, npx reverses the behaviour.

Reply:
script/setup wouldn't need npx, what's the error you see?

Reply:
The error is on `script/test`:
```
> hmpps-accredited-programmes-ui@0.0.1 lint:njk
> npx djlint .

npm ERR! could not determine executable to run
```

But is the `npx` addition needed after running `script/setup`?

Reply:
I skipped script/setup as it runs script/bootstrap, which requires nodenv globally. So far I'm running script/utils/start-backing-services to start docker compose + mocks, and npm run start:dev

Just checking script/test now...

Reply:
[djLint requires Python](https://www.npmjs.com/package/djlint/v/1.6.0#-how) so it might not work too well with npx?

> So far I'm running script/utils/start-backing-services to start docker compose + mocks, and npm run start:dev

We typically run `script/server` to start up the server (we're using the [Scripts To Rule Them All](https://github.com/github/scripts-to-rule-them-all) pattern), which bundles everything you need into one script, rather than running `npm run start:dev` directly

Running `script/setup` initially, `script/update` if you need to bring in any dependency updates (including to Docker images) then `script/server` to load up the server should generally do the trick

Reply:
It failed for me where script/test calls script/update which calls script/bootstrap, which uses the nodenv, another that needs an npx before it.

Reply:
Ahh I see - are you using an alternative to `nodenv`?

Reply:
(nodenv is always installed globally - it's not an npm package)

Reply:
Sorted, I'll leave the package.json alone for now, there's too many commands that I need to begin using first.